ipaserver/plugins/cert.py

```diff
@@ -884,8 +886,8 @@ def execute(self, csr, all=False, raw=False, chain=False, **kw):
                             "match requested principal") % gn.name)
             elif isinstance(gn, cryptography.x509.general_name.RFC822Name):
                 if principal_type == USER:
-                    if principal_obj and gn.value not in principal_obj.get(
```

Comment:
@felipevolpone actually, I was confused.  This whole bit is guarded by an `if principal_type == USER`, and if that is true, then `principal_obj` is set.  So we are actually good to go; the check that was there before your change was
superfluous.